### PR TITLE
[zh] Resync about/deployment/index.md

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
Resync about/deployment/index.md

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/about/deployment/index.md [hunk 1]
```diff
@@ -1,30 +1,138 @@
 ---
 title: 部署
 description: 部署。
-subtitle: Istio 目的是解决当单体应用程序向分布式微服务架构过渡时开发人员和运营商所面临的挑战。
+subtitle: 阅读有关在第 1 天、第 2 天和第 1,000 天快速有效实施的良好实践。
```
Comment (zirain) [inline content/zh/about/deployment/index.md:4]:
```suggestion
subtitle: 阅读有关在第 1 天、第 2 天和第 1000 天快速有效实施的良好实践。
```

File: content/zh/about/deployment/index.md [hunk 2]
```diff
@@ -1,30 +1,138 @@
...
-GKE 的早期实验和 Istio 取得了成功，很快得到了企业的认可。他们几个月来一直试图实现的功能，在短短一周内就突然准备好了。 Istio 不仅能够提供 mTLS，还能够提供健壮的重试和备份策略以及离群值检测。
+{{< text bash >}}
+$ istioctl install
+{{< /text >}}
 
-## 第 3 步{#step-three}
+随着新版本的发布，您可以对其进行测试并逐步将它们推广到您的集群中。
+
+许多托管 Kubernetes 服务提供商都可以选择为您自动安装和管理 Istio。
+查看我们的[分销商页面](/zh/about/ecosystem/)，看看您的供应商是否支持 Istio。
```
Comment (zirain) [inline content/zh/about/deployment/index.md:32]:
```suggestion
查看我们的[供应商页面](/zh/about/ecosystem/)，看看您的供应商是否支持 Istio。
```

File: content/zh/about/deployment/index.md [hunk 3]
```diff
@@ -1,30 +1,138 @@
...
-    - /zh/deployment.html
+  - /zh/deployment.html
 doc_type: about
 ---
-[comment]: <> (TODO: Replace placeholders)
 
 {{< centered_block >}}
 
-## 第 1 步{#step-one}
+您已经决定要使用 Istio。欢迎来到服务网格的世界。祝贺你，你在伟大的公司！
```
Comment (zirain) [inline content/zh/about/deployment/index.md:16]:
```suggestion
您已经决定要使用 Istio。欢迎来到服务网格的世界。祝贺您，您在一家伟大的公司！
```

**Review by zirain — COMMENTED** [review]
small nits, thanks！

File: content/zh/about/deployment/index.md [hunk 4]
```diff
@@ -1,30 +1,138 @@
...
+许多托管 Kubernetes 服务提供商都可以选择为您自动安装和管理 Istio。
+查看我们的[分销商页面](/zh/about/ecosystem/)，看看您的供应商是否支持 Istio。
+
+Istio 还是许多商业服务管理产品的引擎，专家团队随时准备帮助您上手。
+
+有一个不断增长的云原生顾问社区，能够在您的 Istio 之旅中为您提供帮助。
+如果您要与 Istio 生态系统的成员一起工作，我们建议您尽早循环他们。
+我们的许多合作伙伴和分销商已经在这个项目上工作了很长时间，并且在指导您的旅程中将发挥无价的作用。
+
+## 你应该先启用什么？
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:40]:
```suggestion
## 你应该先启用什么？{#what-should-you-enable-first?}
```
Most headings need an anchor like this.

File: content/zh/about/deployment/index.md [hunk 5]
```diff
@@ -1,30 +1,138 @@
...
+许多托管 Kubernetes 服务提供商都可以选择为您自动安装和管理 Istio。
+查看我们的[供应商页面](/zh/about/ecosystem/)，看看您的供应商是否支持 Istio。
+
+Istio 还是许多商业服务管理产品的引擎，专家团队随时准备帮助您上手。
+
+有一个不断增长的云原生顾问社区，能够在您的 Istio 之旅中为您提供帮助。
+如果您要与 Istio 生态系统的成员一起工作，我们建议您尽早循环他们。
+我们的许多合作伙伴和分销商已经在这个项目上工作了很长时间，并且在指导您的旅程中将发挥无价的作用。
+
+## 你应该先启用什么？{#what-should-you-enable-first}
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:40]:
```suggestion
## 您应该先启用什么？{#what-should-you-enable-first}
```
`你`和`您`统一一下吧，建议用`您`

File: content/zh/about/deployment/index.md [hunk 6]
```diff
@@ -1,30 +1,138 @@
...
+
+Istio 不仅适用于 Kubernetes；还可以将虚拟机（或裸机）上的[服务添加到网格](/zh/docs/setup/install/virtual-machine/)中，
+以获得 Istio 提供的所有功能，例如相互 TLS、丰富的遥测和高级流量管理功能。
+
+### 监测你的服务{#monitor-your-services}
+
+使用 [Kiali](/zh/docs/ops/integrations/kiali/) 检查流经你的网格的流量，或者使用 [Zipkin](/zh/docs/tasks/observability/distributed-tracing/zipkin/)
+或 [Jaeger](/zh/docs/tasks/observability/distributed-tracing/jaeger/)追踪请求。
+
+使用 Istio 的默认 [Grafana](/zh/docs/ops/integrations/grafana/) 仪表板，来自动报告在网格中运行的服务的黄金信号。
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:101]:
```suggestion
使用 Istio 的默认 [Grafana](/zh/docs/ops/integrations/grafana/) 仪表板，自动报告在网格中所运行服务的关键信号。
```

File: content/zh/about/deployment/index.md [hunk 7]
```diff
@@ -1,30 +1,138 @@
...
 ---
-[comment]: <> (TODO: Replace placeholders)
 
 {{< centered_block >}}
 
-## 第 1 步{#step-one}
+您已经决定要使用 Istio。欢迎来到服务网格的世界。祝贺您，您在一家伟大的公司！
 
-Istio 解决了开发人员和运营商在单一应用程序向分布式微服务架构过渡时所面临的挑战。了解这一点，有助于更详细地了解 Istio 的服务网格。
+如果您还没有，您可能想在测试环境中试用 Istio，并运行我们的[入门指南](/zh/docs/setup/getting-started/)。
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:18]:
```suggestion
如果您没用过 Istio，可能要在测试环境中进行试用，请参阅我们的[入门指南](/zh/docs/setup/getting-started/)。
```

File: content/zh/about/deployment/index.md [hunk 8]
```diff
@@ -1,30 +1,138 @@
...
-## 第 3 步{#step-three}
+随着新版本的发布，您可以对其进行测试并逐步将它们推广到您的集群中。
+
+许多托管 Kubernetes 服务提供商都可以选择为您自动安装和管理 Istio。
+查看我们的[供应商页面](/zh/about/ecosystem/)，看看您的供应商是否支持 Istio。
+
+Istio 还是许多商业服务管理产品的引擎，专家团队随时准备帮助您上手。
+
+有一个不断增长的云原生顾问社区，能够在您的 Istio 之旅中为您提供帮助。
+如果您要与 Istio 生态系统的成员一起工作，我们建议您尽早循环他们。
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:37]:
```suggestion
如果您要与 Istio 生态系统的成员一起工作，我们建议您尽早熟悉。
```

File: content/zh/about/deployment/index.md [hunk 9]
```diff
@@ -1,30 +1,138 @@
...
+我们的许多合作伙伴和分销商已经在这个项目上工作了很长时间，并且在指导您的旅程中将发挥无价的作用。
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:38]:
```suggestion
我们的许多合作伙伴和供应商已经在这个项目上工作了很长时间，并且在指导您的旅程中将发挥无价的作用。
```

File: content/zh/about/deployment/index.md [hunk 10]
```diff
@@ -1,30 +1,138 @@
...
+## 你应该先启用什么？{#what-should-you-enable-first}
+
+采用 Istio 有很多重要理由：从为微服务增加安全性到提高应用程序的可靠性。
+无论您的目标是什么，最成功的 Istio 实施都是从确定一个用例并解决该用例开始的。
+一旦你配置了网格来解决一个问题，您可以轻松启用其他功能，从而增加部署的实用性。
+
+## 我如何将网格映射到我的架构上？{#how-do-i-map-the-mesh-to-my-architecture}
+
+通过一次添加一个命名空间，逐步将您的服务纳入网格。默认情况下，来自多个命名空间的服务可以相互通信，
+但您可以通过有选择地选择将哪些服务公开给其他命名空间来轻松增加隔离。
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:49]:
```suggestion
但您可以有目的地选择将哪些服务公开给其他命名空间来轻松提高隔离级别。
```

File: content/zh/about/deployment/index.md [hunk 11]
```diff
@@ -1,30 +1,138 @@
...
+
+### 启用安全性{#enabling-security}
+
+Istio 将在网格中配置服务以在相互通信时尽可能使用 mTLS。
+默认情况下，Istio 将以 `permissive mTLS` 模式运行，这意味着服务将接受加密和未加密的流量，以允许来自非网格服务的流量保持功能。
+在所有服务都加入网格后，您可以[改变认证策略，只允许加密流量](/zh/docs/tasks/security/authentication/mtls-migration/)。
+然后您可以确定所有流量都已加密。
+
+Istio 将配置网格中的服务，使其在相互交谈时尽可能使用 mTLS。
+Istio 默认以"允许的 mTLS "模式运行，这意味着服务将同时接受加密和未加密的流量，以允许来自非网格服务的流量保持正常流通。
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:83]:
```suggestion
Istio 默认以"允许的 mTLS" 模式运行，这意味着服务将同时接受加密和未加密的流量，以允许来自非网格服务的流量保持正常流通。
```

File: content/zh/about/deployment/index.md [hunk 12]
```diff
@@ -1,30 +1,138 @@
...
+
+### Istio 的两类 API{#istio's-two-types-of-apis}
+
+Istio 为平台所有者和服务所有者提供 API。根据您扮演的角色，您只需要考虑一个子集。
+例如，平台所有者将拥有安装、认证和授权资源。流量管理资源将由服务所有者处理。[了解哪些 API 对你有用](/zh/docs/reference/config/)。
+
+### 在虚拟机上连接服务{#connect-services-on-virtual-machines}
+
+Istio 不仅适用于 Kubernetes；还可以将虚拟机（或裸机）上的[服务添加到网格](/zh/docs/setup/install/virtual-machine/)中，
+以获得 Istio 提供的所有功能，例如相互 TLS、丰富的遥测和高级流量管理功能。
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:94]:
```suggestion
以获得 Istio 提供的所有功能，例如 mTLS、丰富的遥测和高级流量管理功能。
```

File: content/zh/about/deployment/index.md [hunk 13]
```diff
@@ -1,30 +1,138 @@
...
+
+当发布新版本时，Istio 允许就地升级和金丝雀升级。
+在两者之间进行选择，是在简单性和潜在停机时间之间进行权衡。
+对于生产环境，推荐使用[金丝雀升级方式](/zh/docs/setup/upgrade/canary/)。
+在新的控制和数据平面版本被验证工作后，您可以升级您的网关。
+
+### 监控网格{#monitor-the-mesh}
+
+Istio 为网格内的所有服务通信生成详细的遥测数据。
+这些指标、跟踪和访问日志对于了解您的应用程序如何相互交互以及识别任何性能瓶颈至关重要。
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:124]:
```suggestion
这些指标、链路和访问日志对于了解您的应用程序如何交互以及识别所有性能瓶颈至关重要。
```

File: content/zh/about/deployment/index.md [hunk 14]
```diff
@@ -1,30 +1,138 @@
...
+使用此信息可帮助您设置熔断器、超时和重试并强化您的应用程序。
+
+就像在网格中运行的应用程序一样，Istio 控制平面组件也会导出指标。
+利用这些指标和预配置的 Grafana 仪表板来调整您的资源请求、限制和扩展。
+
+## 加入 Istio 社区{#join-the-Istio-community}
+
+一旦您运行了 Istio，您就成为了一个大型全球社区的成员。您可以在[我们的讨论论坛](https://discuss.istio.io/)或
```
Comment (windsonsea) [inline content/zh/about/deployment/index.md:132]:
```suggestion
一旦您运行了 Istio，您就成为了一个大型全球社区的成员。您可以在[我们的论坛](https://discuss.istio.io/)或
```

**Review by windsonsea — APPROVED** [review]
lgtm

**Pull request merged**